foundry.toml

```diff
@@ -0,0 +1,13 @@
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ['lib']
+test = 'test'
+script = 'script'
+fs_permissions = [{ access = "read", path = "./.env" }]
+remappings = [
+    "@openzeppelin/=lib/openzeppelin-contracts/",
+    "forge-std/=lib/forge-std/src/",
+]
+
```

Comment:
I wonder if we'd better lock the Solidity compiler version by setting `solc_version` here to keep consistency across builds.

Reply:
We can make a PR change and upgrade the current contract version to 0.8.20 , This will change too much.